### PR TITLE
Feature: Added-contacts-file

### PR DESCRIPTION
**Pull request by Clemond**
In this PR:
1. Added contacts file

### CODE REVIEW

**Review by hampus-code — COMMENTED** [review]
this looks good!

**Review by carolCase — COMMENTED** [review]
greate work!

**Review by hampus-code — APPROVED** [review]
this looks good again!

**Review by ahmadardal — COMMENTED** [review]
It would be nice to know which restaurant these contacts are for, perhaps add a headline for the contact of each restaurant?

File: contacts.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+It would be nice to know which restaurant these contacts are for, perhaps add a headline for the contact of each restaurant?
+
```
Comment (ahmadardal) [inline contacts.txt:3]: Lol, what happened here?

**Review by carolCase — APPROVED** [review]

File: contacts.txt (hunk 1 above)
Comment (Clemond) [reply contacts.txt]: Sry, fixed now :D

**Review by ahmadardal — APPROVED** [review]

**Pull request merged**